docs/pages/Configuration.md

```diff
@@ -702,7 +702,7 @@ A list of bundle definitions. A `bundleDefinition` contains of the following opt
     - `declareRawModules`: Whether raw modules should be declared after jQuery.sap.global became available. With the usage of the ui5loader, this flag should be set to 'false'. By default set to `false`
     - `renderer`: Whether renderers for controls should be added to the module set. By default set to `false`
     - `sort`:  By default, modules are sorted by their dependencies. The sorting can be suppressed by setting the option to `false`
-    - `async`: Specifies whether the require section of the module should be asynchronous. When set to `true`, the modules will be loaded using sap.ui.require instead of sap.ui.requireSync. The latter API is not available in UI5 version 2.x. **Note:** This property is available only for `mode=require`
+    - `async`: Specifies whether the require section of the module should be asynchronous. When set to `true`, the modules will be loaded using sap.ui.require instead of sap.ui.requireSync. The latter API is not available in UI5 version 2.x. **Note:** This property is available only for `mode=require` and defaults to `true`
```

Comment:
Not related to this change, but when reviewing the file as a while I noticed that the nesting doesn't seem to be right. "sections" should be on the same level as "defaultFileTypes", shouldn't it? Strangely, it looks still okay in the V4 documentation [^1], but not when previewing the MD file in the code section of the Github WebUI [^2]

[^1]: https://sap.github.io/ui5-tooling/v4/pages/Configuration/#custom-bundling
[^2]: https://github.com/SAP/ui5-tooling/blob/main/docs/pages/Configuration.md

Reply:
Thanks, @codeworrior !

Indeed, there was indentation before the "sections" item and GH treats this as subitem.
I fixed it with https://github.com/SAP/ui5-tooling/pull/970